### DISCUSSION TITLE
Question about sensor entities in v 0.8.0

### DISCUSSION THREAD

**Discussion by jkyselak** [blakeblackshear/frigate#632] [Ask A Question] [answered]
First of all: Brilliant piece of software!

I had version 0.7.3 running pretty smooth and was looking forward to 0.8.0.
But there I ran into some issues with the sensors, especially the binary ones that triggered a notification for me.
So, my question is: Do I still add the sensors to the HA configuration.yaml? I noticed the docs do not mention it anymore.
When I delete the sensors from the configuration.yaml, none of them are available. Even though I have the hass integration installed via HACS.
I know it's probably something stupid but any help would be appreciated

**Comment by blakeblackshear** [comment, marked as answer]
Make sure you actually setup the integration under Configuration > Integrations in Homeassistant. HACS just makes it available in that list.

**Reply by jkyselak** [reply]
That was the missing piece!
Thank you so much for your help!
